tgui-next/packages/tgui/interfaces/ShuttleManipulator.js

```diff
@@ -20,39 +20,39 @@ export const ShuttleManipulator = props => {
           <Section>
             <table>
               {shuttles.map(shuttle => (
-                <tr key={shuttle.id + '_row'}>
-                  <td key={shuttle.id + '_jmprow'}>
+                <tr key={shuttle.id}>
+                  <td key={shuttle.id}>
```

Comment:
Oy, I have just noticed you assign a key to all of the elements. Assign keys only to top level array elements, i.e. `tr`, rest don't matter. Linter will help you.

Reply:
make sense